DOC-3013: Modifications to Bulk Email "Send To" Groups; allow multi-select

catong opened the pull request:
This PR updates bulk email procedures for course teams to reflect new addressing options in bulk email, including ability to select more than one of the groups.
It does not include ability to select cohorts as a "send to" option - this change will be covered in a separate PR when the code merges.

Reviewers:
- [x] @efischer19 
- [x] @lamagnifica 

FYI @sstack22, @jaakana

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -27,14 +27,13 @@ This section contains the following topics.
 
-When you send an email message from the instructor dashboard, you choose its
-recipients by selecting one of these predefined groups.
+When you send an email message from the Instructor Dashboard, you choose its
```
lc instructor dashboard

catong replied:
Ack: a merge conflict resolution loss!

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -27,14 +27,13 @@ This section contains the following topics.
-When you send an email message from the instructor dashboard, you choose its
-recipients by selecting one of these predefined groups.
+When you send an email message from the Instructor Dashboard, you choose its
+recipients by selecting a **Send to** option. If applicable, you can select
```
maybe just, "You can select one or more of the following recipients for each message"

catong replied:
I had already modified the wording to point out that you can select multiple options only if it makes sense - e.g. if you select a specific cohort you can't also select "all learners", but then removed the cohort selection piece from this PR. I'll change the wording so that it's oriented more towards selecting recipients than selecting options.

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -27,14 +27,13 @@ This section contains the following topics.
-* **All (learners, staff and admins)**, to communicate with currently
-  enrolled learners and the course team.
+* **Staff and admins**, to contact other members of the course team.
+* **All learners**, to communicate with all currently enrolled learners.
```
does this still include staff and admins? or does it exclude staff and admins, but now you can check both check boxes if you want to send to them both?

efischer19 replied:
As I understand it, it is possible to be a staff member and *not* be enrolled in the course, which is what the `Staff and admins` box is for. The `All learners` options is "everyone who is enrolled in the course" which may (but doesn't have to) include instructors.

To ensure sending to `All` (as the previous option existed), you'd need to select both options. Duplicates are filtered out, so a staff user who is also enrolled would still only receive the message once.

efischer19 replied:
Maybe something like:

**Staff and admin**, to contact other members of the course team whether they are enrolled or not.
**All learners**, to communicate with all currently enrolled learners.

And add something about duplicate filtering?

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -44,10 +43,10 @@ recipients by selecting one of these predefined groups.
-canceled** after they are sent. If you plan to send a message to all course
-participants, be sure to review each draft carefully, and send the message to
-yourself first for thorough testing.
+.. note:: Before you use the bulk email feature, consider that messages cannot
```
maybe even make this an "important"?

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -44,10 +43,10 @@ recipients by selecting one of these predefined groups.
-yourself first for thorough testing.
+.. note:: Before you use the bulk email feature, consider that messages cannot
+   be cancelled after they are sent. Before sending a message, be sure to
+   review each draft carefully, and send the message to yourself first for
```
^only to yourself? (meaning, just select Myself and no other possible recipients at least once...)

lamagnifica commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -111,12 +110,16 @@ To send an email message to course participants, follow these steps.
 
-#. Select who you want to send the message to from the **Send to** drop-down
-   list. You select from the following options.
+#. Select email recipients from the **Send to** list. If applicable, you can
```
list --> options

efischer19 replied:
This comment still hasn't been addressed

lamagnifica commented:
question for @efischer19 about what "all learners" means now. 
Otherwise, a few suggestions to use as make sense.
👍

efischer19 commented on en_us/shared/manage_live_course/bulk_email.rst:
```diff
@@ -111,12 +110,16 @@ To send an email message to course participants, follow these steps.
+   select more than one option.
+
+  * **Myself**
+  * **Staff and admins**
```
**Staff and admin**, if we're matching the screenshots exactly.

Should I change this in code, to keep plurality consistent between `Staff and admin` and `All learners`?

catong replied:
@efischer19 That would be ideal - thanks! If it's not too long, "Staff and Administrators" would be best, otherwise "Staff and Admins".

lamagnifica replied:
"Admin" is how the role name appears on the LMS membership page and Studio Course Team page....

efischer19 replied:
I was going to suggest `Admin` over `Administrator` as well, for spacing reasons. Is the pluralization (`Admins`) still desired?

<img width="261" alt="screen shot 2016-06-03 at 2 07 23 pm" src="https://cloud.githubusercontent.com/assets/7373924/15788245/8c0f532a-2994-11e6-8d88-7a1af1ff4fb5.png">

catong replied:
@efischer19 Our notes crossed (see my comment about role names vs descriptive noun for people with those roles in the main conversation page). To be truly parallel I still think it should be "Administrators" but if space does not allow, "Admins" (yes please to the plural) is fine.

catong commented:
@lamagnifica and @efischer19 Thanks for your input. I've addressed Alison's comments and updated the label strings per Eric's comments (I used "Staff and Administrators" for now; I can update if the final string is not that).

catong commented:
@efischer19 @lamagnifica A further note about the role names. In the platform we often conflate the names of the roles with nouns describing people with the roles. In past work I've tried to make the difference more clear in doc and UI, plus for translation and globalization reasons, I don't like "Admins". IMO I don't think it's confusing to use "Staff and Administrators" in the Send to field even if our actual role name is "Admin", but I'm open to suggestions otherwise.

efischer19 commented:
👍 from me, if the "list --> options" comment from above was ignored deliberately. Looks great!

catong commented:
@efischer19 Nope it wasn't deliberate - thanks for the catch!

catong commented:
Merging https://github.com/edx/edx-documentation/pull/1058
